dev/RadioButtons/RadioButtons.cpp

```diff
@@ -65,6 +65,13 @@ void RadioButtons::OnApplyTemplate()
         return static_cast<winrt::ItemsRepeater>(nullptr);
     }());
 
+    const auto name = winrt::AutomationProperties::GetName(*this);
+    if (name.empty())
+    {
+        // If the automation name isn't set, use header text as the name.
+        winrt::AutomationProperties::SetName(*this, SharedHelpers::TryGetStringRepresentationFromObject(Header()));
+    }
+
```

Comment:
I think we should also do this on property changed for our header so it's staying up to date. Also could/should do it when the AutiomationProperties.Name property changes.

Reply:
I realized it wasn't really the right way to do it -- the header text should be returned from the automation peer. So this is all fixed up now. :)